web/components/Navbar.tsx

```diff
@@ -23,8 +23,10 @@ const useStyles = makeStyles({
   },
 });
 
-type NavbarProps = {};
-export const Navbar: React.FC<NavbarProps> = () => {
+type NavbarProps = {
+  user: any;
```

Comment:
are we going to declare types down the line?

Reply:
hmm. I think in order to get these types we'd need to generate them for the graphql schema. Maybe we should tackle that first. WDYT?

Reply:
Done in #34